### PR TITLE
RESOPS-956:Allow .email domains in emails

### PR DESCRIPTION
**Pull request by blackcathacker**

### CODE REVIEW

File: v2_1/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    In addition, it will also return an appropriate HTTP status code
+    (i.e. `500` if there was an error; `200` if all went well).
+
+## XML Schema Versioning
+
+1.  Version numbers will be recorded using both major and minor numbers:
+
+    ```
+      <schema version="1.0"
+        targetNamespace="https://github.com/rSmart/ce-tech-docs/tree/master/v1_0">
```
Comment (leoherbie) [inline v2_1/README.md]: please change the github url references to rSmart

File: v2_1/hrmanifest.xsd [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+  </xs:simpleType>
+
+  <!--
+  +=======================+====================================+
+  | CITIZENSHIP_TYPE_CODE | DESCRIPTION                        |
+  +=======================+====================================+
+  |                     1 | US CITIZEN OR NONCITIZEN NATIONAL  |
+  |                     2 | PERMANENT RESIDENT OF US           |
+  |                     3 | NON US CITIZEN WITH TEMPORARY VISA |
+  |                     4 | PERMANENT RESIDENT OF US PENDING   |
```
Comment (leoherbie) [inline v2_1/hrmanifest.xsd]: might want to update with code 15 & 16

File: v2_1/hrmanifest.xsd [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+  |                     5 | OTHER                              |
+  +=======================+====================================+ -->
+  <xs:simpleType name="citizenshipType">
+    <xs:restriction base="xs:int">
+      <xs:totalDigits value="1"/>
+      <xs:minInclusive value="1"/>
+      <xs:maxInclusive value="5"/>
```
Comment (leoherbie) [inline v2_1/hrmanifest.xsd]: since we support 1-5 & 15, 16 we have to do this as an enum type

Comment (blackcathacker) [inline v2_1/hrmanifest.xsd]: Hmm...how about just max being 16. That means the xsd might validate potentially invalid values, but those will all be caught when the xml is uploaded? I'm wondering if we want to relax this even more as each institution can add any citizenship type they want and seems like the xsd should at least allow those?

Comment (blackcathacker) [inline v2_1/hrmanifest.xsd]: 1 - 1 trillion-ish like the table allows? `CITIZENSHIP_TYPE_CODE | decimal(15,0)`? Or maybe just 1-100?

**Pull request merged**